include/Core/EmailLogger.php

```diff
@@ -30,41 +30,40 @@ public function log_email( $mail_info ) {
 		 * @since Genesis
 		 *
 		 * @param array $mail_info {
-		 *     @type string $to
+		 *     @type string|array $to
 		 *     @type string $subject
 		 *     @type string $message
-		 *     @type string $headers
-		 *     @type string $attachment
+		 *     @type string|array $headers
+		 *     @type string|array $attachment
 		 * }
 		 */
 		$mail_info = apply_filters( 'el_wp_mail_log', $mail_info );
 
 		// Sometimes the array passed to the `wp_mail` filter may not contain all the required keys.
 		// See https://wordpress.org/support/topic/illegal-string-offset-attachments/
-		$mail_info = wp_parse_args( $mail_info, array(
-			'attachments' => array(),
+		$mail_info_replica = wp_parse_args( $mail_info, array(
 			'to'          => '',
```

Comment:
We need to include `attachment` also

Reply:
@mariadanieldeepak Any specific reason for removing `attachment` key?

Reply:
@sudar,

Fixed the above comment in the following commit.

https://github.com/sudar/email-log/commit/480983556e57a56b285b659d6a23e2b9a390c571